src/ui/server/constants/routes/insurance/policy/loss-payee.ts

```diff
@@ -0,0 +1,29 @@
+import POLICY_ROOT from './root';
+
+const ROOT = `${POLICY_ROOT}/loss-payee`;
+const LOSS_PAYEE_DETAILS_ROOT = `${POLICY_ROOT}/loss-payee-details`;
+const LOSS_PAYEE_FINANCIAL_UK_ROOT = `${POLICY_ROOT}/loss-payee-bank-details`;
+const LOSS_PAYEE_FINANCIAL_INTERNATIONAL_ROOT = `${ROOT}/loss-payee-international-bank-details`;
+
+const LOSS_PAYEE_ROUTES = {
+  LOSS_PAYEE_ROOT: ROOT,
+  LOSS_PAYEE_SAVE_AND_BACK: `${ROOT}/save-and-back`,
+  LOSS_PAYEE_CHANGE: `${ROOT}/change`,
+  LOSS_PAYEE_CHECK_AND_CHANGE: `${ROOT}/check-and-change`,
+  LOSS_PAYEE_DETAILS_ROOT,
+  LOSS_PAYEE_DETAILS_CHANGE: `${LOSS_PAYEE_DETAILS_ROOT}/change`,
+  LOSS_PAYEE_DETAILS_CHECK_AND_CHANGE: `${LOSS_PAYEE_DETAILS_ROOT}/check-and-change`,
+  // TODO: remove ROOT from everything apart from the the root
```

Comment:
should these TODOs be addressed in this PR or a later one?

Reply:
This will be in the next PR, half complete locally. Will change a lot of files.